tests/build.test.js

```diff
@@ -1,64 +1,70 @@
-import { resolve } from 'path';
-import fs from 'fs.promised';
-import htmlLooksLike from 'html-looks-like';
+import { join } from 'path';
+import { readFile } from 'fs.promised';
+import looksLike from 'html-looks-like';
 import { create, build } from './lib/cli';
-import lsr from './lib/lsr';
-import { setup, fromSubject } from './lib/output';
-import expectedOutputs, { sassPrerendered, withCustomTemplate, multiplePrerenderingHome, multiplePrerenderingRoute } from './build.snapshot';
-import filesMatchSnapshot from './lib/filesMatchSnapshot';
+import { snapshot, isMatch } from './lib/utils';
+import { fromSubject } from './lib/output';
+import images from './images/build';
+
+// TODO
+// const ours = ['empty', 'full', 'simple', 'root'];
+const ours = ['default', 'full'];
```

Comment:
It should be `default` & `simple` here too! (coz we can run tests on those templates)

Reply:
Same as above ^^